src/api/controller/MyController.hpp

```diff
@@ -302,53 +303,58 @@ class MyController : public oatpp::web::server::api::ApiController {
     if (!auth.tokenCheck(authObject->token)) {
       responseDto->success = false;
       return createDtoResponse(Status::CODE_401, responseDto);
-    } else {
-      std::vector<bakcyl::core::Item> items;
-      for (const auto& pair : *json->data) {
-        auto& itemsList = pair.second;
-        for (const auto& itemJson : *itemsList) {
-          auto item = bakcyl::core::Item{itemJson->whoBuy->c_str(),
-                                         itemJson->item->c_str(),
-                                         itemJson->price, itemJson->amount};
-          items.push_back(item);
-        }
-      }
+    }
 
-      auto friends = receiptOper.calculateReceipt(items);
+    if (!json || !json->data || !json->receiptName) {
+      responseDto->success = false;
+      return createDtoResponse(Status::CODE_400, responseDto);
+    }
 
-      if (friends.empty()) {
-        responseDto->success = false;
-        return createDtoResponse(Status::CODE_400, responseDto);
+    std::vector<bakcyl::core::Item> items;
+    for (const auto& pair : *json->data) {
```

Comment:
for (const auto& [idDontKnowWhatIsUnderFirstItem_createApropriateVaraibleName, itemList] : json->data)

Reply:
impossible to do because the oat++ library does not allow it